herbarium-sheet-plant-organ-detection/main.py

```diff
@@ -11,18 +12,19 @@
 from detectron2.config import get_cfg
 from detectron2.engine.defaults import DefaultPredictor
 from detectron2 import model_zoo
-from typing import Tuple
+from typing import Tuple, Any, Dict, List
```

Comment:
Hmm yes it is a bit difficult to fully cover it. So for not all Dicts we would fully know what is in there. Let's say for the DigitalSpecimen the value might be a str, int, float or another Dict with in it the same things. So that is where we say `Any` so we know the key is a string, but the value could be anything.